.github/workflows/deploy-to-pantheon.yml

```diff
@@ -45,6 +45,12 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  warn_on_version:
+    name: "Warn that support for v0 ending"
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "::error title=Update now::Support for the download/upload-artifact@v3 actions are ending in November 2024. This means v0 of this workflow will stop working. You must update to the v1 version of this workflow to continue using it."
```

Comment:
Does this make workflows error out if people are using this action? Is there a way for people to turn that off if they're not ready to update yet?

Not a huge deal as I expect not many people are using this, mostly curious about the effects of the `::error`.

Reply:
Confusingly, it doesn't! It puts an error annotation on the Summary screen of the workflow run. It looks like so:

![image](https://github.com/softwareforgood/reusable-github-workflows/assets/94169/ec68b250-ed9e-4295-a05c-3d20245f64bc)

Reply:
To be clear, it doesn't error out the workflow run though.